Thing.h

```diff
@@ -695,10 +695,10 @@ class ThingDevice {
 #endif
   }
 
-  void serialize(JsonObject descr, String ip, uint16_t port) {
+  void serialize(JbsonObject descr, String ip, uint16_t port) {
     descr["id"] = this->id;
     descr["title"] = this->title;
-    descr["@context"] = "https://webthings.io/schemas";
+    descr["@context"] = "  https://www.w3.org/2019/wot/td/v1";
```

Comment:
The correct value should be: `['https://www.w3.org./2019/wot/td/v1','https://webthings.io/schemas'].